app/(profile)/my-profile/container/Container.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { PropsWithChildren } from 'react'
+import styles from './Container.module.scss'
+
```

Comment:
**nitpick:** Why the same component is duplicated? We should have one container for centering web app content